sdk/storage/AzureStorageBlob/Source/Operation/ResumableOperationQueue.swift

```diff
@@ -34,13 +34,11 @@ internal protocol ResumableOperationQueueDelegate: ResumableOperationDelegate {
 internal class ResumableOperationQueue {
     // MARK: Properties
 
-    internal lazy var operations: [ResumableOperation] = []
+    public lazy var operationQueue: OperationQueue = OperationQueue()
 
-    internal lazy var operationQueue: OperationQueue = OperationQueue()
+    weak var delegate: ResumableOperationQueueDelegate?
```

Comment:
FYI, I want to get rid of ResumableOperationQueueDelegate entirely and let the operations directly call `transfer(didUpdate:...)` instead of an `operation(didUpdate:...)` that then signals the transfer. However, I thought that would be too much in this PR and wasn't necessary to fix the bug.